ArduPlane/control_modes.cpp

```diff
@@ -159,34 +159,54 @@ void Plane::reset_control_switch()
 /*
   called when entering autotune
  */
-void Plane::autotune_start(void)
+void Plane::autotune_start(uint8_t bitmask)
 {
     gcs().send_text(MAV_SEVERITY_INFO, "Started autotune");
-    rollController.autotune_start();
-    pitchController.autotune_start();
-    yawController.autotune_start();
+    if(bitmask == 0){
+        bitmask = 7 ;
+    }
+    if(1&bitmask){
```

Comment:
The interpretation of this bitmask should be done in the mavlink layer, not in here.

What should be passed into here is probably a bitmask made up of `AP_AutoTune::ATType` members.